### ISSUE TITLE
init() function (timer 1 configuration) fails on ATtiny45 / 85. [imported]

### ISSUE THREAD

**Issue by cmaglie** [arduino/Arduino#496]
This is [Issue 496](http://code.google.com/p/arduino/issues/detail?id=496) moved from a Google Code project.
Added by 2011-03-05T18:11:21.000Z by [dmel...@gmail.com](http://code.google.com/u/107726014984682000895/).
Please review that bug for more context and additional comments, but update this bug.
 Closed (Fixed).

Original labels: Type-Defect, Priority-Medium, Component-Core, Milestone-1.0
### Original description

**What steps will reproduce the problem?**
1.Create a ATtiny45/85 version of the pins_arduino.h file.
2.Try to compile a program.

**What is the expected output? What do you see instead?**

You get "wiring.c:215: error: 'TCCR1B' undeclared".  

Basically, the timer initialization in the init() function isn't modular / portable enough.  We'll probably need to move something into pins_arduino.h, or add more #ifdefs in init().
